computer_vision/haralick_descriptors.py

```diff
@@ -0,0 +1,434 @@
+"""
+https://en.wikipedia.org/wiki/Image_texture
+https://en.wikipedia.org/wiki/Co-occurrence_matrix#Application_to_image_analysis
+"""
+from typing import Any
+
+import imageio.v2 as imageio
+import numpy as np
+
+
+def root_mean_square_error(original: np.ndarray, reference: np.ndarray) -> float:
+    """Simple implementation of Root Mean Squared Error
+    for two N dimensional numpy arrays.
+
+    Examples:
+        >>> root_mean_square_error(np.array([1, 2, 3]), np.array([1, 2, 3]))
+        0.0
+        >>> root_mean_square_error(np.array([1, 2, 3]), np.array([2, 2, 2]))
+        0.816496580927726
+        >>> root_mean_square_error(np.array([1, 2, 3]), np.array([6, 4, 2]))
+        3.1622776601683795
```

Comment:
Over indenting doctests disables them.
```suggestion
    >>> root_mean_square_error(np.array([1, 2, 3]), np.array([1, 2, 3]))
    0.0
    >>> root_mean_square_error(np.array([1, 2, 3]), np.array([2, 2, 2]))
    0.816496580927726
    >>> root_mean_square_error(np.array([1, 2, 3]), np.array([6, 4, 2]))
    3.1622776601683795
```